src/invidious/yt_backend/youtube_api.cr

```diff
@@ -341,6 +341,10 @@ module YoutubeAPI
       client_context["client"]["platform"] = platform
     end
 
+    if CONFIG.visitor_data.is_a?(String)
+      client_context["client"]["visitorData"] = CONFIG.visitor_data.as(String)
+    end
+
```

Comment:
I'm not sure what is the point of that addition? Client-specific visitor data? But it's never used.

Reply:
it's required to pass both poToken and visitorData for poToken to work.

I told about it on the internal matrix room.